client/src/redux/actions/twitterActions.js

```diff
@@ -1,11 +1,43 @@
 import axiosInstance from '../../api/buildClient';
 import {
   SET_ERRORS,
+  TWITTER_ADD_STREAM,
   TWITTER_RESET_PROFILE_INFO,
   TWITTER_SET_HOME_TIMELINE_TWEETS,
   TWITTER_SET_PROFILE_INFO,
+  TWITTER_SET_STREAM_LOADING_STATUS,
+  TWITTER_SET_STREAM_TWEETS,
+  TWITTER_UPDATE_STREAMS,
   USER_SET_MESSAGES,
 } from '../types';
+import { store } from '../store';
+
+const handleError = ({ error }) => async (dispatch) => {
+  if (
+    error &&
```

Comment:
This is a great use case for the optional chaining operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Fixed